frontend/src/component/project/Project/PaginatedProjectFeatureToggles/useProjectFeatureSearch.ts

```diff
@@ -61,3 +66,42 @@ export const useProjectFeatureSearch = (
         setTableState,
     };
 };
+
+export const useProjectFeatureSearchActions = (
+    tableState: ReturnType<typeof useProjectFeatureSearch>['tableState'],
+    setTableState: ReturnType<typeof useProjectFeatureSearch>['setTableState'],
+) => {
+    const onAttributeClick = (attribute: Attribute, value: string) => {
```

Comment:
click tag/feature type and have identical behavior. Use default operator with current value if no previous value is set. Otherwise keep the operator and append the value to the list.